components/FormToCreateShoppingItem.js

```diff
@@ -0,0 +1,108 @@
+import styled from "styled-components";
+
+export default function FormToCreateShoppingItem({ onAddItem, categories }) {
+  console.log("Categories: ", categories);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const newItem = Object.fromEntries(formData);
+    newItem.quantity = Number(newItem.quantity);
+    newItem.imageUrl = "placeholder_1.png";
+
+    onAddItem(newItem);
+    event.target.reset();
+  }
+
+  return (
+    <article>
+      <container>
```

Comment:
There's no HTML element called "container" and it doesn't seem to fulfill any purpose here. "article" should be enough

Reply:
Right. Thanks for pointing this out.